system/lib/libc/musl/arch/emscripten/bits/syscall.h

```diff
@@ -1,132 +1,3 @@
-/*
- * The .h version of this file is generated from the .h.in.
```

Comment:
so this file is not actually generated now?

Reply:
Yes exactly.. we no longer need the duplication so we no longer need to tool to help us maintain it.